addons/account-settings-capitalize/style.css

```
@@ -1,4 +1,5 @@
 .account-nav .dropdown > li:nth-child(3),
-ul.user-nav > li:nth-child(3) {
+ul.user-nav > li:nth-child(3), .account-nav .dropdown > li:nth-child(3),
```

i think this is supposed to be
```suggestion
ul.user-nav > li:nth-child(3), .account-nav .dropdown > li:nth-child(4),
```
?